File: EPS BackEnd/EPS.WebServices/EPS.Common.Models/Medicine/MedicineRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EPS.Common.Models
{
    public class MedicineRequest
    {
        public string UserCode { get; set; }
        /// <summary>
        /// Historia
        /// </summary>
        public int History { get; set; }
        /// <summary>
        /// Medicamento
        /// </summary>
        public Medicine Medicine { get; set; }
    }
}

File: EPS FrontEnd/EPS/src/app/models/Medicine/medicine-request.ts
import { Medicine } from './medicine';

///Parametros peticion Medicamento
export class MedicineRequest
{
  UserCode: string;
  /// <summary>
  /// Historia
  /// </summary>
  History: number;
  /// <summary>
  /// Medicamento
  /// </summary>
  Medicine: Medicine;
}
